Source/WebInspectorUI/UserInterface/Views/WebInspectorExtensionTabContentView.js

```diff
@@ -78,6 +78,11 @@ WI.WebInspectorExtensionTabContentView = class WebInspectorExtensionTabContentVi
         return `ExtensionTab-${this._extension.extensionBundleIdentifier}-${this._tabInfo.displayName}`;
     }
 
+    set iframeURL(sourceURL)
+    {
+        this._iframeElement.src = sourceURL;
```

Comment:
I'm pretty sure setting the `src` to the same value will still trigger a load.  Do we want that?  Or should we bail if it's the same URL?

Reply:
Yeah, all cases we are calling this we want the extension to load the URL fresh.